src/connectors/relevant-sort/__tests__/connectRelevantSort-test.ts

```diff
@@ -277,11 +277,11 @@ describe('connectRelevantSort', () => {
 
       expect(
         widget.getWidgetUiState!(
-          { relevantSort: { relevancyStrictness: 25 } },
+          { relevantSort: 25 },
           { helper, searchParameters: helper.state }
         )
       ).toEqual({
-        relevantSort: { relevancyStrictness: undefined },
+        relevantSort: 25,
```

Comment:
I don't understand why it was 25 and not undefined before, this seems like a fix

Reply:
When the helper state doesn't include appliedRelevancyStrictness at all, then the widget shouldn't, either. I guess that's what this test is about.

Reply:
but if the helper state doesn't have relevancy strictness, but the ui state (route) does have it, doesn't that apply the strictness in getWidgetSearchParameters?